composables/use-reader.ts

```diff
@@ -1,12 +1,21 @@
-export default function () {
+export default function (
+  {
+    nftClassIdInput,
+  }: {
+    nftClassIdInput?: Ref<string> | string
+  } = {},
+) {
```

Comment:
For consistency, can we keep `nftClassId` across composables? then use `params.nftClassId`
```suggestion
export default function (params: {
  nftClassId?: Ref<string> | string
} = {}) {
```

Reply:
[🎨 Update Arweave endpoint and simplify parameter naming](https://github.com/likecoin/3ook-com/pull/47/commits/ba8def248f66aa90a8583bd483940f68fd008603)